python/ray/experimental/state/state_cli.py

```diff
@@ -82,7 +85,7 @@ def list_state_cli_group(ctx):
         namespace=ray_constants.KV_NAMESPACE_DASHBOARD,
         num_retries=20,
     )
-
+    api_server_url = api_server_url.decode()
```

Comment:
Isn't it raising an exception is `api_server_url ` is None?

Reply:
oh damn - sorry, my bad.